src/cc65/codeseg.c

```diff
@@ -426,8 +426,10 @@ static CodeEntry* ParseInsn (CodeSeg* S, LineInfo* LI, const char* L)
         unsigned Hash = HashStr (Arg) % CS_LABEL_HASH_SIZE;
         Label = CS_FindLabel (S, Arg, Hash);
 
-        /* If we don't have the label, it's a forward ref - create it */
-        if (Label == 0) {
+        /* If we don't have the label, it's a forward ref - create it unless
+        ** it's an external function.
+        */
+        if (Label == 0 && (OPC->OPC != OP65_JMP || IsLocalLabelName (Arg)) ) {
```

Comment:
I'm confused by commit dafac1d4bd4663dc4fa9b35f55e5413100c7e22f.  The commit message says, "Only for jumps".  But, that opcode test creates labels for anything _except_ jumps!  Is that test wrong; or, do I misunderstand the message?